static/js/clients.js

```diff
@@ -0,0 +1,143 @@
+document.addEventListener('DOMContentLoaded', (event) => {
+    // Display filters only if javascript is enabled
+    let filtersBar = document.getElementById("filters");
+    filtersBar.style.display = "flex";
```

Comment:
We can do this differently to avoid flickering on page load: Just add a `<noscript>` tag in the `head` that contains a stylesheet that hides the element.